Cargo.toml

```diff
@@ -68,6 +68,7 @@ hashbrown = ">= 0.12, <0.13"
 dashmap = ">= 5.4, <5.5"
 inferno = ">= 0.11, < 0.11.15"
 pprof = { version = "0.9", features = ["criterion", "flamegraph"] } # MSRV 1.56
+backtrace = "=0.3.67" # Last version required rust 1.65
```

Comment:
Done, but I had to add two new pins because even Rust 1.65 can't compile the latest version:

    1. Pinned blake2b_simd to "1.0.1" as its latest version requires Rust 1.66.
    2. Pinned clap to "4.2.0" as its latest version requires Rust 1.70.

Additionally, I've added a couple of comments with "FIXME" to draw your attention to the lines that were updated in Cargo.toml in the latest version of Orchard from the Zcash team.

Reply:
Also, I've updated the dependencies in Cargo.toml to align with the latest changes from the Zcash team's upstream Cargo.toml.